wiki/Tournaments/112/Summer_2019/en.md

```diff
@@ -0,0 +1,656 @@
+| :-- | :-- | :-- |
+| #1 | ![][flag_TR] [pyth0nix](https://osu.ppy.sh/users/10922849) | 2 |
+| #2 | ![][flag_TR] [mezelyus](https://osu.ppy.sh/users/5938859) | 2,6 |
+| #3 | ![][flag_TR] [Fequin](https://osu.ppy.sh/users/11665593) | 3,6 |
+| #4 | ![][flag_TR] [Tsuda-](https://osu.ppy.sh/users/6713666) | 4,8 |
+| #5 | ![][flag_TR] [tingirmin](https://osu.ppy.sh/users/9027514) | 5,4 |
+| #6 | ![][flag_TR] [Purple_Glasses](https://osu.ppy.sh/users/9045874) | 6 |
+| #7 | ![][flag_TR] [withmystyle](https://osu.ppy.sh/users/7953720) | 8,2 |
+| #8 | ![][flag_TR] [empirasu](https://osu.ppy.sh/users/12500084) | 9 |
+| #9 | ![][flag_TR] [emptypudding](https://osu.ppy.sh/users/11003085) | 9,4 |
+| #10 | ![][flag_TR] [Ekrem Imamoglu](https://osu.ppy.sh/users/10801545) | 10,8 | 
+| #11 | ![][flag_TR] [sezaiemre](https://osu.ppy.sh/users/10807430) | 11,4 |
+| #12 | ![][flag_TR] [aimlessplayr](https://osu.ppy.sh/users/4903088) | 11,4 |
+| #13 | ![][flag_TR] [HayunGuys](https://osu.ppy.sh/users/9693852) | 13,4 |
+| #14 | ![][flag_TR] [amdinstartor](https://osu.ppy.sh/users/10505284) | 13,4 |
+| #15 | ![][flag_TR] [karyuunatsu](https://osu.ppy.sh/users/12575190) | 14 |
+| #16 | ![][flag_TR] [Horoz](https://osu.ppy.sh/users/11614447) | 14 |
+| E | ![][flag_TR] [L Lones](https://osu.ppy.sh/users/4954493) | 14,6 |
+| E | ![][flag_TR] [Mikazu](https://osu.ppy.sh/users/12474487) | 17 |
+
+Only first 16 players from each seed are qualified, the rest is eliminated.
+
+---
+
+## Teams
+
+After Group matches, those who get through to Quarterfinals are obliged to form a team with a player from the other seed.
+
+| Team Name | 1 - 15K Player | 15K+ Player |
+| :-: | :-- | :-- | 
+| KaFein | ![][flag_DE] [-Kanzaki](https://osu.ppy.sh/users/1652478) | ![][flag_TR] [Fequin](https://osu.ppy.sh/users/11665593) |
+| bum patladın | ![][flag_TR] [Raikouhou](https://osu.ppy.sh/users/8007528) | ![][flag_TR] [Tsuda-](https://osu.ppy.sh/users/6713666) |
+| Kwk Yelleri | ![][flag_TR] [KaraElmas](https://osu.ppy.sh/users/9300293) | ![][flag_TR] [mezelyus](https://osu.ppy.sh/users/5938859) | 
+| KFC Managers | ![][flag_TR] [dragonsaga](https://osu.ppy.sh/users/4982690) | ![][flag_TR] [pyth0nix](https://osu.ppy.sh/users/10922849) |
+| eTi dasmjasdk | ![][flag_TR] [edizberkserbest](https://osu.ppy.sh/users/9256771) | ![][flag_TR] [tingirmin](https://osu.ppy.sh/users/9027514) |
+| egemen ve aimless | ![][flag_TR] [egemenbsrms](https://osu.ppy.sh/users/4520477) | ![][flag_TR] [aimlessplayr](https://osu.ppy.sh/users/4903088) |
+| İsim bulamadık pingleme artık | ![][flag_TR] [AkashiDagara](https://osu.ppy.sh/users/6118485) | ![][flag_TR] [empirasu](https://osu.ppy.sh/users/12500084) |
+| Yürü baba fingo | ![][flag_TR] [Iron Man](https://osu.ppy.sh/users/4242381) | ![][flag_TR] [emptypudding](https://osu.ppy.sh/users/11003085) |
+
+---
+
+![112ST 2019 Podium](img/Podium.png)
+
+---
+
+## Mappools
+
+### Grand Finals
+
+**[Download the map pack here!](https://112.s-ul.eu/yUBjoWpe)**
```

Comment:
```suggestion
**[Download the map pack here!](https://112.s-ul.eu/yUBjoWpe)**
```
You can also hint the expected file size
e.g. [Download the map pack here! (60 MB)]